wg-docs-tools/README.md

```diff
@@ -17,6 +17,16 @@ Oversees externally-focused tooling (e.g. Fiddle, Forge), Electron documentation
 | <img src="https://github.com/nitsakh.png" width=100 alt="@nitsakh">  | Nitish Sakhawalkar [@nitsakh](https://github.com/nitsakh) | Member | PT (San Francisco) |
 | <img src="https://github.com/shiftkey.png" width=100 alt="@shiftkey">  | Brendan Forster [@shiftkey](https://github.com/shiftkey) | Member | AT (Canada) |
 
+## Current Objective and Key Results
+**Objective:**
+
+As a newcomer or experienced app developer, you can use Electron’s tooling and documentation to build your app well.
+
+**Key Results:**
+* Increase adoption of Electron-provided tooling.
+* Decrease cases of documented poor-practices - normalized against the randomness and size of the samples
```

Comment:
I'm not quite clear on what this means?

Reply:
Could be rephrased. The intent was to keep a small sample size of respondents from causing the KR to pass/fail due to statistical noise.

Part of the discussion for this KR was that @felixrieseberg goes around and talks to different Electron app devs and asks them about their experience. That's a good feedback channel, but if we define success based on that sample alone, the KR could pass/fail just based on the luck of the draw of who Felix talked to that quarter.

Reply:
@electron/wg-docs-tools 
I was not at the post-summit meeting. Can someone from this WG clarify this?

Reply:
ah, gotcha! i'd love to make perhaps elaborate a bit more to make this a bit clearer  :)

Reply:
ah, gotcha! i'd love to make perhaps elaborate a bit more to make this a bit clearer  :)